tests/views/schema.py

```diff
@@ -108,6 +108,10 @@ def root_name(self) -> str:
     def value_from_context(self, info: strawberry.Info) -> str:
         return info.context["custom_value"]
 
+    @strawberry.field
+    def value_from_extensions(self, key: str, info: strawberry.Info) -> str:
```

Comment:
nit: info first, to follow the pattern of others that receive `info` as the first one, but this one receives an extra `key`

Maybe also adding a `, *, ` in the middle to enforce `key` to be passed as a keyword would be good?

```suggestion
    def value_from_extensions(self, info: strawberry.Info, key: str) -> str:
```